tutorials/ner_double/README.md

```diff
@@ -0,0 +1,114 @@
+<!-- SPACY PROJECT: AUTO-GENERATED DOCS START (do not remove) -->
+
+# 🪐 spaCy Project: Combining Multiple Trained NER Components
+
+This project shows you the different ways you can combine multiple trained NER components and their tradeoffs.
+Note that before running this project, you should run the [ner_drugs](../ner_drugs) project and install the packaged version of it.
+
+
+## 📋 project.yml
+
+The [`project.yml`](project.yml) defines the data assets required by the
+project, as well as the available commands and workflows. For details, see the
+[spaCy projects documentation](https://spacy.io/usage/projects).
+
+### ⏯ Commands
+
+The following commands are defined by the project. They
+can be executed using [`spacy project run [name]`](https://spacy.io/api/cli#project-run).
+Commands are only re-run if their inputs have changed.
+
+| Command | Description |
+| --- | --- |
+| `assemble` | Build two versions of the combined model from configs. |
+| `package` | Package the models so they can be installed. |
+| `visualize-model` | Use the model interactively using Streamlit |
```

Comment:
I think in v3 we'd use the term "pipeline" here everywhere instead of "model". We've kind of changed that terminology in comparison to v2, as now the ML models in the pipeline components are more explicitly defined in the constructor, and things got confusing ;-)

Reply:
Makes perfect sense.